docs/reference/commandline/search.md

```diff
@@ -19,6 +19,69 @@ parent = "smn_cli"
+    peelsky/zulu-openjdk-busybox                                                     1                    [OK]
+    skomma/busybox-data              Docker image suitable for data volume cont...   1                    [OK]
+    elektritter/busybox-teamspeak    Leightweight teamspeak3 container based on...   1                    [OK]
+    socketplane/busybox                                                              1                    [OK]
+    oveits/docker-nginx-busybox      This is a tiny NginX docker image based on...   0                    [OK]
+    ggtools/busybox-ubuntu           Busybox ubuntu version with extra goodies       0                    [OK]
+    nikfoundas/busybox-confd         Minimal busybox based distribution of confd     0                    [OK]
+    openshift/busybox-http-app                                                       0                    [OK]
+    jllopis/busybox                                                                  0                    [OK]
+    swyckoff/busybox                                                                 0                    [OK]
+    powellquiring/busybox                                                            0                    [OK]
+    williamyeh/busybox-sh            Docker image for BusyBox's sh                   0                    [OK]
+    simplexsys/busybox-cli-powered   Docker busybox images, with a few often us...   0                    [OK]
+    fhisamoto/busybox-java           Busybox java                                    0                    [OK]
+    scottabernethy/busybox                                                           0                    [OK]
+    marclop/busybox-solr
+
+This example will display images with the name containing busybox.
+
+### Search images by limiting stars' number(-s OR --stars)
+    
+    $ docker search --stars 3 busybox
+    NAME                 DESCRIPTION                                     STARS     OFFICIAL   AUTOMATED
+    busybox              Busybox base image.                             325       [OK]       
+    progrium/busybox                                                     50                   [OK]
+    radial/busyboxplus   Full-chain, Internet enabled, busybox made...   8                    [OK]
+
+This example will display images with the name containing busybox and at least 3 stars.
+
+### Search automated images(--automated)
+
+    $ docker search --stars=3 --automated=true busybox
+    NAME                 DESCRIPTION                                     STARS     OFFICIAL   AUTOMATED
+    progrium/busybox                                                     50                   [OK]
+    radial/busyboxplus   Full-chain, Internet enabled, busybox made...   8                    [OK]
+
+This example will display images with at least 3 stars and automated builds.
+
+### Fully display 'DESCRIPTION'(--no-trunc=true)
```

Comment:
I'd prefer to skip `=true` for boolean options.

Perhaps change the title to: "Display non-truncated descriptions (--no-trunc)"  (button to better suggestions)